lake-manifest.json

```diff
@@ -37,18 +37,6 @@
     "rev": "72610cec1cd686884a08704f5fa91b45262136f9",
     "name": "aesop",
     "inputRev?": "master"}},
-  {"git":
```

Comment:
Why are these changes being made in this PR?

Reply:
They should not be -- this was in error.